File: nightgrass/src/main/java/com/goodfeel/nightgrass/dto/OrderDto.java
package com.goodfeel.nightgrass.dto;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

import java.math.BigDecimal;
import java.time.Instant;
import java.time.LocalDateTime;

@Getter @Setter @AllArgsConstructor @NoArgsConstructor
public class OrderDto {
    private Long orderId;
    private String orderNo;
    private String userId;
    private String deliveryAddress;
    private BigDecimal orderTotal;
    private Instant createdAt;
    private LocalDateTime orderProcessDate;
    private String logisticsNo;
    private LocalDateTime deliveryDate;
    private String orderStatus;
    private String payNo;
    private String payType;
    private String remark;
}

File: nightgrass/src/main/kotlin/com/goodfeel/nightgrass/dto/OrderDto.kt
package com.goodfeel.nightgrass.dto

import com.goodfeel.nightgrass.util.OrderStatus
import java.math.BigDecimal
import java.time.LocalDateTime

data class OrderDto(
    val orderId: Long,
    val orderNo: String,
    val userId: String,
    val deliveryAddress: String? = null,
    val originalTotal: BigDecimal,
    val discountedTotal: BigDecimal,
    val hst: BigDecimal,
    val shippingFee: BigDecimal,
    val orderTotal: BigDecimal,
    val createdAt: LocalDateTime,
    val orderProcessDate: LocalDateTime? = null,
    val logisticsNo: String? = null,
    val deliveryDate: LocalDateTime? = null,
    val orderStatus: OrderStatus
)
